src/storage/storage.py

```diff
@@ -1,21 +1,21 @@
 """
-src/storage/storage.py
-======================
+Storing inventory items
 """
 import collections
 
-InventoryItem = collections.namedtuple('InventoryItem', 'type stream payload expires tag')
+InventoryItem = collections.namedtuple(
+    'InventoryItem', 'type stream payload expires tag')
 
 
-class Storage(object):
-    """Base class for storing inventory (extendable for other items to store)"""
+class Storage(object):  # pylint: disable=too-few-public-methods
```

Comment:
@g1itch What if we remove this class?

Reply:
better ask @PeterSurda, I have no complete understanding of this module. If you are worried about pylint `too-few-public-methods` concider adding `__metaclass__ = ABCMeta`.

Reply:
It appeared in 36b5e2c

Reply:
I think the term for this is abstract class. Maybe there is a way to write it in a more pythonic way, such as defining the methods but have them all contain just `pass`

Reply:
No need to define methods because parent class already defined them